airflow/operators/python.py

```diff
@@ -145,6 +146,8 @@ class _PythonDecoratedOperator(BaseOperator):
     """
 
     template_fields = ('op_args', 'op_kwargs')
+    template_fields_renderers = {"op_args": "py", "op_kwargs": "py"}
```

Comment:
I hope no one will be angry that I added this in this PR 😄 

<img width="737" alt="Screenshot 2020-11-10 at 18 10 38" src="https://user-images.githubusercontent.com/9528307/98707216-1466ce80-2380-11eb-8b00-c01fb8f11494.png">